api/types.proto

```diff
@@ -459,12 +459,26 @@ enum TaskState {
 	// Remove this message when the states are deemed perfect.
 }
 
+// State of Networks attached to a container
+message ContainerNetworkStatus {
+	// Network is the Network ID
+	string network = 1;
+
+	// Interface is the device name as seen within the container
+	string interface = 2;
+
+	// Address is the IP address in the form ipv4:XXX or ipv6:YYY
+	string address = 3;
```

Comment:
Good point, I did indeed make this up myself.

`api.EndPoint.VirtuallIP.addr` is a string. Looking at the usage it appears to be defined as one which is compatible with `net.ParseCIDR`/`net.ParseIP`. This seems like a reasonable enough precedent to follow.